modules/node_modules/@oncojs/react-lolliplot/LolliplotNode.js

```diff
@@ -175,6 +176,17 @@ let Lolliplot = ({
     })
     .on(`mousedown`, d => onPointClick(d, d3.event))
 
+    svg
+    .append(`g`)
+    .selectAll(`text`)
+    .data(data)
+    .enter()
+    .append(`text`).attrs({
+      x: d => scaleLinear(d.x),
+      y: d => scaleLinearY(d.y),
+      fill: 'black'
+    }).text(d => {return collisions[`${d.x},${d.y}`] ? collisions[`${d.x},${d.y}`].length : '' })
```

Comment:
pretty sure this can be just

```
.text(d => collisions[`${d.x},${d.y}`] || collisions[`${d.x},${d.y}`].length)
```

unless it shows `undefined` in which case the ternary is still needed